frontend/src/services/CodeEditorService.js

```diff
@@ -0,0 +1,12 @@
+// latest list of language_versions taken from https://emkc.org/api/v2/piston/runtimes
```

Comment:
Perhaps this could be in utils folder instead of services since this file is a constants file

Reply:
noted !